tableaudocumentapi/datasource.py

```diff
@@ -234,16 +235,15 @@ def clear_repository_location(self):
         if tag is not None:
             self._datasourceXML.remove(tag)
 
-    ###########
-    # fields
```

Comment:
I think these added more clutter than they helped, and weren't consistently applied.

I left the section headers in place (Public API, Private API)